docs/ProfilerOptions.md

```diff
@@ -0,0 +1,91 @@
+| `--total`                                          | `total`                                      | Count the total value of the collected metric instead of the number of samples, e.g. total allocation size.                                                                                                                                                                                                                                                                                                                                          |
+| `--chunksize N`, `--chunktime N`                   | `chunksize=N`, `chunktime=N`                 | Approximate size and time limits for a single JFR chunk. A new chunk will be started whenever either limit is reached. The default `chunksize` is 100MB, and the default `chunktime` is 1 hour.</br>Example: `asprof -f profile.jfr --chunksize 100m --chunktime 1h 8983`                                                                                                                                                                            |
+| `-I include`, `-X exclude`                         | `include=PATTERN`, `exclude=PATTERN`         | Filter stack traces by the given pattern(s). `-I` defines the name pattern that *must* be present in the stack traces, while `-X` is the pattern that *must not* occur in any of stack traces in the output. `-I` and `-X` options can be specified multiple times. A pattern may begin or end with a star `*` that denotes any (possibly empty) sequence of characters.</br>Example: `asprof -I 'Primes.*' -I 'java/*' -X '*Unsafe.park*' 8983`     |
+| `-L level`                                         | `loglevel=level`                             | log level: `debug`, `info`, `warn`, `error` or `none`.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `-F features`                                      | `features=LIST`                              | Comma separated list of HotSpot-specific features to include in stack traces. Supported features are:<ul><li>`vtable` - display targets of megamorphic virtual calls as an extra frame on top of `vtable stub` or `itable stub`.</li><li>`comptask` - display current compilation task (a Java method being compiled) in a JIT compiler stack trace.</li></ul>                                                                                       |
```

Comment:
+pcaddr #1001

Reply:
Addressed in [56300c5](https://github.com/async-profiler/async-profiler/pull/1029/commits/56300c529468865dcf38189dab464e1fff07e352)